src/hw1/Main.java

```diff
@@ -0,0 +1,325 @@
+package hw1;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+// CoworkingSpace: represents a coworking space
+class CoworkingSpace {
```

Comment:
please separate all this classes from Main, in Main.java should be only Main class.